inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -237,8 +245,15 @@ MKLDNNPadNode::PadExecutor::PadExecutor(const PadAttrs& attrs,
                                         const VectorDims& srcDims,
                                         const VectorDims& dstDims) {
     params.attrs = attrs;
-    params.srcDims = srcDims;
     params.dstDims = dstDims;
+
+    cornerCase = std::any_of(srcDims.begin(), srcDims.end(), [](size_t dim) { return dim == 0; } ) &&
```

Comment:
What behavior should be for modes other than CONSTANT? Maybe throw exception?

Reply:
As far as I understood from reference implementation we should fill it with zero value.
Choosing pad value: https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/op/pad.cpp#L201
Filling: https://github.com/openvinotoolkit/openvino/blob/master/src/core/reference/src/runtime/reference/pad.cpp#L64